pkgs/applications/virtualization/oci-seccomp-bpf-hook/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, buildGoModule
+, fetchFromGitHub
+, go-md2man
+, installShellFiles
+, libseccomp
+, linuxPackages
+, pkg-config
+}:
+
+buildGoModule rec {
+  pname = "oci-seccomp-bpf-hook";
+  version = "1.2.0";
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "oci-seccomp-bpf-hook";
+    rev = "v${version}";
+    sha256 = "143x4daixzhhhpli1l14r7dr7dn3q42w8dddr16jzhhwighsirqw";
+  };
+  vendorSha256 = null;
+  badPlatforms = [ "aarch64-linux" ];
```

Comment:
@zowoq is this enough to exclude the architecture? :innocent:

Reply:
I think `badPlatforms` needs to be in `meta` with `platforms`.

Reply:
Hah, thank you for the hint! This one was new to me :)